website/docs/r/slack_connection.html.markdown

```diff
@@ -94,7 +94,7 @@ The following attributes are exported:
 
 ## Import
 
-Slack connections can be imported using the related `workspace` ID and the `slack_connection` ID separated by a dot, e.g.
+Slack connections can be imported using the related `slack_team(workspace)` ID and the `slack_connection` ID separated by a dot, e.g.
```

Comment:
Would you mind reverting this particular change please? because here We are only referring to the names of the attributes 🙏🏽